### PR TITLE
Update README.md for the Java Voice Assistant quickstart and fix broken links

### PR DESCRIPTION
**Pull request by dargilco**

### CODE REVIEW

**Comment by dargilco** [comment]
@jhakulin please review. I fill remove the "en-us" in the docs.microsoft link in a new PR. Thanks!

**Comment by dargilco** [comment]
@jhakulin please review. Thanks!

File: quickstart/java/jre/virtual-assistant/README.md [hunk 1]
```diff
@@ -56,5 +54,5 @@ See the [accompanying article](https://docs.microsoft.com/azure/cognitive-servic
 
 ## References
 
-* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-virtual-assistant-java-jre)
-* [Speech SDK API reference for Java](https://aka.ms/csspeech/javaref)
+* [Quickstart article on the SDK documentation site](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-voice-assistant-java-jre)
```
Comment (BrianMouncer) [inline quickstart/java/jre/virtual-assistant/README.md]: >en-us [](start = 80, length = 5)

i think docs guys will flag this, as your suppose to let the browser choose the users language.

File: quickstart/java/jre/virtual-assistant/README.md [hunk 2]
```diff
@@ -56,5 +54,5 @@ See the [accompanying article](https://docs.microsoft.com/azure/cognitive-servic
...
+* [Speech SDK API reference for Java](https://docs.microsoft.com/en-us/java/api/com.microsoft.cognitiveservices.speech.dialog?view=azure-java-stable)
```
Comment (BrianMouncer) [inline quickstart/java/jre/virtual-assistant/README.md]: >en-us/ [](start = 65, length = 6)

same here.

File: quickstart/java/jre/virtual-assistant/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
-# Quickstart: Using virtual assistant (Preview) in Java on Windows or Linux
+# Quickstart: Using Voice Assistant in Java on Windows or Linux
 
-This sample demonstrates how to recognize speech for custom voice-first virtual assistants (Preview) and receive activity responses.
-See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-virtual-assistant-java-jre) on the SDK documentation page which describes how to build this sample from scratch in Eclipse.
+This sample demonstrates how to recognize speech for custom voice assistants and receive activity responses.
+See the [accompanying article](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/quickstart-voice-assistant-java-jre) on the SDK documentation page which describes how to build this sample from scratch in Eclipse.
```
Comment (BrianMouncer) [inline quickstart/java/jre/virtual-assistant/README.md]: >en-us [](start = 58, length = 5)

same...is this needed.

File: quickstart/java/jre/virtual-assistant/README.md (hunk 2 above)
Comment (BrianMouncer) [reply quickstart/java/jre/virtual-assistant/README.md]: Darren will fix the Language tag in URL in his next PR...

**Pull request merged**